views/partials/prs.hbs

```diff
@@ -1,3 +1,9 @@
+{{#exists openedPrs}}
+<div class="f6 ph2 tc flex-none">
+    <p class="white">PRs already open for 2018: <strong>{{ openedPrs }}</strong>.</p>
```

Comment:
Can you change the wording slightly... `PRs opened so far this month:` Also can we put the openedPrs through a number format, you can either do this in the node (probably easiest) or use a handlebars helper like https://gist.github.com/DennyLoko/61882bc72176ca74a0f2.

Reply:
Sure :+1: